reactioncommerce/reaction
# Avalara Tax Reporting

Closes #1704 

Uses TaxRate API to estimate taxes for cart
Record orders as "SalesInvoice" for Tax Reporting

Test plan:


With Avalara not enabled, ensure that you can checkout w/o issues (ensuring that address validation doesn't interfere)

1. Enable Avalara with key, username and password
1. Ensure that taxes are calculated correctly in cart
1. Complete an order
1. Login to admin panel (info in Wiki) and ensure that a "SalesInvoice" is recorded for that order
1. Refund that order
1. Ensure that a return is recorded

Testing Address validation:

1. Try a valid US address
1. Try an invalid US address
1. Try a valid Canadian Address (valid here means that the place actually exists)
1. Try an invalid Canadian address
1. Try an address from an unsupported country (PH is one)
